pkg/image/controller/scheduled_image_controller.go

```diff
@@ -189,6 +197,13 @@ func resetScheduledTags(stream *imagev1.ImageStream) {
 func needsScheduling(stream *imagev1.ImageStream) bool {
 	for _, tagRef := range stream.Spec.Tags {
 		if tagImportable(tagRef) && tagRef.ImportPolicy.Scheduled {
+			if ref, err := imageref.Parse(tagRef.From.Name); err != nil && len(tagRef.From.Name) > 0 {
+				return false
+			} else if len(ref.ID) > 0 && !tagNeedsImport(stream, tagRef, true) {
+				// ref.ID is set if this is a canonical, sha/digest ref;
+				// we allow scheduled import of those, but only until the initially succeed
+				return false
```

Comment:
What if we have two tags and the second one needs to be scheduled?

Reply:
good catch @dmage 

I'll queue up a follow up PR momentarily.

Reply:
yes, thanks @dmage, this should be a continue like is in resetScheduledTags.  (same for the err check above)